### ISSUE TITLE
Monero Community Workgroup Meeting: December 7th 16:00 UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#1120]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://www.getmonero.org/resources/user-guides/join-monero-matrix.html)

Time
16:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
    - [RandomX 5 year anniversary](https://www.reddit.com/r/Monero/comments/1h6e4nk/randomx_5_year_anniversary/)
    - Bi-Directional XMR<>BCH swaps added to BasicSwapDex - [tweet](https://xcancel.com/BasicSwapDEX/status/1862646285657866396)
    - FCMP++ video - Vostoemisio [reddit](https://www.reddit.com/r/Monero/comments/1h80qnr/fullchain_membership_proofs_fcmp_explainer_video/)
News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/)
5. [CCS updates](https://ccs.getmonero.org/)    
  a [v1docq47 - monerotopia 2024 voiceovers and working on xmr.ru](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/517)    
  b. [selsta part-time monero development (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/522)    
  c. Kewbits Haveno GUI [update / partial payout request](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/489#note_27599)
  d. [CryptoCheckout Email marketing campaign to onboard new merchants](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/520)
6. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2025](https://monerokon.org)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
7. Open ideas time    
8. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/)    

Meeting logs will be posted here afterwards.    

not discussing / voted to close:
e. [Carrot animated video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/506)    
f. [CryptoCheckout WordPress plugin (for WooCommerce) & Shopify app](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/514)

**Comment by plowsof** [comment]
Logs 
> __< plowsof >__ kewbit will be unbanned prior to the meeting, further discussion about haveno related things will be moved to haveno related rooms prior and after the meeting     

> __< plowsof >__ sharing opinion on what to handle the situation is welcomed though     

> __< 3​21bob321:monero.social >__ God i hope this ends after that     

> __< r​ottenwheel:unredacted.org >__ No, plowsof will let it drag forever a la mbll. 😂     

> __< p​ubichair:matrix.org >__ The new dart guy was… 🤏     

> __< p​ubichair:matrix.org >__ Small penis     

> __< r​ottenwheel:unredacted.org >__ https://matrix.monero.social/_matrix/media/v1/download/unredacted.org/tRdsiALuQUQdPfwifSBXeQFx     

> __< s​yntheticbird:monero.social >__ good morning     

> __< p​rivacyx:monero.social >__ I think we should give kewbit to at least up to the meeting today before enacting any bans     

> __< s​yntheticbird:monero.social >__ plowsof just said kewbit will be unbanned prior to the meeting     

> __< r​ottenwheel:unredacted.org >__ I think he attempted to release a blog post recently but took it down almost immediately. My RSS feed bot sent me this:     

> __< r​ottenwheel:unredacted.org >__ > Kewbit Died     

> __< r​ottenwheel:unredacted.org >__ > https://kewbit.org/untitled/     

> __< 3​21bob321:monero.social >__ https://kewbit.org/dead/     

> __< 3​21bob321:monero.social >__ Sad     

> __< a​remor:matrix.org >__ Good points. It’s just reto still has incredible spreads most of the time still == not quite mature yet. Any additional networks will create more fragmentation. Right now might not be the best time for another network.  Reto is growing fairly well; with a few more months of growth, more networks would then make sense IMO     

> __< p​lowsof:matrix.org >__ #haveno:monero.social #haveno-lounge:monero.social     

> __< a​remor:matrix.org >__ Yes. I never was for nagging any exception     

> __< p​rivacyx:monero.social >__ https://matrix.monero.social/_matrix/media/v1/download/monero.social/jFTEkIBbWGldhfEZXbbYEGiL     

> __< p​rivacyx:monero.social >__ https://matrix.monero.social/_matrix/media/v1/download/monero.social/dRkfUWBQjikyevhWWqMaRsDX     

> __< p​rivacyx:monero.social >__ I thought you were joking     

> __< p​rivacyx:monero.social >__ I dont get this kid wtf is he doing very immature     

> __< s​yntheticbird:monero.social >__ sir, there are np kids on matrix/irc     

> __< s​yntheticbird:monero.social >__ sir, there are no kids on matrix/irc     

> __< sneurlax_ >__ in Zack Fox's seminal work, "uhhh", he explains: "Red dot on a nigga cheek, he look like Pikachu"[1] ([1] genius.com)     

> __< sneurlax_ >__ he wants to be a pokemon like me     

> __< sneurlax_ >__ but as Mr. Fox so clearly delineates in just the very next line: "All my niggas evil look like people off of Beetlejuice"     

> __< sneurlax_ >__ kewbit is a new pokemon at best--later generation     

> __< sneurlax_ >__ im an og, here to stay     

> __< sneurlax_ >__ IN CONCLUSION--     

> __< sneurlax_ >__ I apologize for being silly     

> __*__ sneurlax_ stops the saturday morning cartoons     

> __< sneurlax_ >__ oh, and for irc-only context, if you're not visiting kewbit.org/dead, you don't know that kewbit's website shows a pikachu.  i thought that kewbit's avatar was a pokemon or other creature I'd seen previously, but upon second look I can't find it (I'm not wasting time researching pokemon, please believe me, dear reader) so my references may not     

> __< sneurlax_ >__ check out regarding this matter     

> __< o​frnxmr:xmr.mx >__ Pichu     

> __< o​frnxmr:xmr.mx >__ or smthn     

> __< o​frnxmr:xmr.mx >__ did you see his blog post after the "dead" one tho?     

> __< o​frnxmr:xmr.mx >__ https://kewbit.org/elyon/     

> __< o​frnxmr:xmr.mx >__ I didnt read this ofc     

> __< sneurlax_ >__ yes, I've been to Elyon.  I keep getting my internet turned off, I think the kewbit """feds""" are onto me     

> __< sneurlax_ >__ oh,I mean, no I haven't ... ofc ...     

> __< p​reland:monero.social >__ Just ran zerogpt on elyon; it looks like the first 3 or so paragraphs are AI, but after that point it’s human     

> __< sneurlax_ >__ oh, I just read the beginning blurb available on the front page, lol     

> __< p​reland:monero.social >__ What’s odd is that the (definitely not a parable) story implies that he is somehow going to prove to the community that he is in the right and that the “elders” need to change     

> __< p​reland:monero.social >__ Which would only make sense if he plans to release source     

> __< o​frnxmr:xmr.mx >__ He was paid 75xmr and pulls bs like haveno.com and https://codeberg.org/HavenoDEX     

> __< o​frnxmr:xmr.mx >__ If he releases code on https://codeberg.org/HavenoDEX, who did he prove wrong?     

> __< s​yntheticbird:monero.social >__ gentlemen lets wait for the meeting     

> __< s​yntheticbird:monero.social >__ keep the ammo in the closet at the moment     

> __< p​reland:monero.social >__ Speaking of change, maybe we should add some sort of “no excessive drama” rule for new CCSs?     

> __< sneurlax_ >__ whomst amongst us hasn't reinvested milestone payments into sweet domains     

> __< sneurlax_ >__ we're all among friends here, we can be honest.     

> __< p​reland:monero.social >__ (And the reason I’m speaking on it early is because I’ll be missing the entire meeting; my IRL has somehow been perfectly planned such that I miss every single Monero-related meeting)     

> __< o​frnxmr:xmr.mx >__ Dont need a rule for common sense     

> __< p​lowsof:matrix.org >__ im starting to think this respectful request for 50% upfront may not have the communities support     

> __< sneurlax_ >__ whomst amongst us hasn't squatted 12 sweet package/crate names     

> __< o​frnxmr:xmr.mx >__ Kewbit is currently banned for trying to stir up drama for hours leading up to the meeting     

> __< o​frnxmr:xmr.mx >__ @sneurlax cast the first stone     

> __< o​frnxmr:xmr.mx >__ package/crate names/git orgs/websites/code     

> __< p​reland:monero.social >__ If he somehow releases all of the code, then that means that he *technically* fulfills the CCS. However, this comes at the expense of a lot of recent meaningless drama that wasted a lot of people’s time. I don’t think that should be tolerated moving forward     

> __< o​frnxmr:xmr.mx >__ Sorry syn, i'll wait     

> __< s​iren:kernal.eu >__ The CEO has arrived     

> __< o​frnxmr:xmr.mx >__ I dont think it should be tolerated moving backward either     

> __< p​reland:monero.social >__ It *shouldn’t* be, but we already agreed to specific terms for the current CCSs and adding extra terms would be unprofessional.      

> __< p​reland:monero.social >__ You know, because trying to bend the contract terms isn’t a good thing (cough cough)     

> __< o​frnxmr:xmr.mx >__ This isnt a government courthouse where you get off on a technicality. This is a community run project and anyone trying to take advantage of Monero and its contributors/community, should be made an example of     

> __< sneurlax_ >__ what's the harm of a little hot goss'     

> __*__ sneurlax_ pounds fist on desk, starts to rise     

> __< sneurlax_ >__ whomst amongst us--     

> __< o​frnxmr:xmr.mx >__ When you get merged for a ccs, then proceed to squat dpmains, crates, code, and extort, play the victim etc. The terms of your ccs dont matter. and in this case, kewbit didnt have donors     

> __< s​iren:kernal.eu >__ I always thought domain hoarders/brokers/squatters were awful people     

> __*__ sneurlax_ blushes     

> __< s​yntheticbird:monero.social >__ They are, I knew a bunch     

> __< o​frnxmr:xmr.mx >__ I would tend to agree that, if people donated to kewbit, that kewbit's ccs had some protection due to people donating for it. But his ccs was 100% repurposed funds. 0 donors     

> __< sneurlax_ >__ i have a second amendment right to hoard domains!     

> __< s​yntheticbird:monero.social >__ Can you keep the ammo in the closet     

> __< s​yntheticbird:monero.social >__ just wait an hour     

> __< p​lowsof:matrix.org >__ meeting in 33 minutes https://github.com/monero-project/meta/issues/1120     

> __< o​frnxmr:xmr.mx >__ Can i load it? I wont shoot, promise     

> __< sneurlax_ >__ geez you try and play it cool and professional but people just come and snipe you for a little domain hoarding misunderstanding     

> __< p​lowsof:matrix.org >__ a proposal for an email campaign promoting a payment processor that isn't going to be merged was also made https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/520     

> __< sneurlax_ >__ why isn't a spam email campaign going to be merged?     

> __*__ sneurlax_ puts on his robe and wizard hat for some ccs magic     

> __< o​frnxmr:xmr.mx >__ same person as https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/514     

> __< sneurlax_ >__ rino-wallet/monero-javascript, huh?  fishy.  doesn't have my woodser seal of authenticity on it     

> __< p​lowsof:matrix.org >__ what have you done to sneurlax and is he ok?     

> __< sneurlax_ >__ this is sneurlax hanging chad version     

> __< s​yntheticbird:monero.social >__ cartoons and its consequences has been a disaster for human race     

> __< sneurlax_ >__ a certified organic* woodser seal     

> __< l​m:matrix.baermail.fr >__ releasing the code is not enough, the code need to works too. I tried the lib he wrote for 75xmr and I could not get it working. No working  example either as of right now. I'm a bit new to Dart/Flutter so maybe I missed something.     

> __< l​m:matrix.baermail.fr >__ https://github.com/KewbitXMR/flutter_haveno/issues/2     

> __< o​frnxmr:xmr.mx >__ He promptly left the room when asked why it doesnt work     

> __< plowsof >__ thanks for testing this lm     

> __< sneurlax_ >__ (rino-wallet/monero-javascript is just forked from woodser/monero-ts minus 309 commits, btw.)     

> __< o​frnxmr:xmr.mx >__ Nvm. He left as soon as he know LM was going to try it     

> __< o​frnxmr:xmr.mx >__ Privacyx also said that he couldn't get the apk to work. I have apk but obv never installed kt     

> __< p​lowsof:matrix.org >__ sneurlax that whole repo is a mess, random pdf's of certificates, even i can see that 'a human must have wrote this!' and generally low quality     

> __< o​frnxmr:xmr.mx >__ Privacyx is the one who opened the bounty iirc     

> __< p​lowsof:matrix.org >__ for cryptoswap* and its 3~ years old     

> __< sneurlax_ >__ a woocommerce plugin for the monero_c php bindings would be sweet     

> __< l​m:matrix.baermail.fr >__ releasing the code is not enough, the code need to works too. I tried the lib he wrote for 75xmr and I could not get it working. No working  example exist as of right now. I'm a bit new to Dart/Flutter so maybe I missed something.     

> __< l​m:matrix.baermail.fr >__ https://github.com/KewbitXMR/flutter\_haveno/issues/2     

> __< sneurlax_ >__ I like issue #1 there, too     

> __< o​frnxmr:xmr.mx >__ too bad its all fake     

> __< o​frnxmr:xmr.mx >__ Going to use txkeys to submit to a block explorer to do a tx_proof 🫠     

> __< p​reland:monero.social >__ That would fit with what the unknown dm person told me     

> __< o​frnxmr:xmr.mx >__ Xmr has 9 atomic units like solana or smthn     

> __< sneurlax_ >__ 🤦     

> __< sneurlax_ >__ ofrnxmr, don't tell me he's not cool     

> __< p​lowsof:matrix.org >__ oh yes i forgot about the atomic units error lol     

> __< p​lowsof:matrix.org >__ if plowsof can audit your work and spot errors, then its not a good luck     

> __< p​lowsof:matrix.org >__ look*     

> __< p​lowsof:matrix.org >__ barely literate^     

> __< o​frnxmr:xmr.mx >__ Are we there yet     

> __< plowsof >__ Meeting time https://github.com/monero-project/meta/issues/1120     

> __< plowsof >__ greetings all!     

> __< plowsof >__ birthday wishes to RandomX which celebrates its 5th https://www.reddit.com/r/Monero/comments/1h6e4      

> __< l​m:matrix.baermail.fr >__ 404     

> __< s​yntheticbird:monero.social >__ hello, happy birthday randomX     

> __< m​alori:xavi.lu >__ happi birthday  randomX :3     

> __< plowsof >__ sorry, correct link https://www.reddit.com/r/Monero/comments/1h6e4nk/randomx_5_year_anniversary/     

> __< plowsof >__ [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/)     

> __< plowsof >__ any other none haveno-related news since the last meeting? 😅 Revuo looks packed with some recent events      

> __< plowsof >__ rucknium.me/donate is functional again after nioCat reported outdated rates, had to navigate docker with him and then update the backend with latest binaries and such , got there in the end      

> __< plowsof >__ logs of the last MRL meeting @ https://github.com/monero-project/meta/issues/1119     

> __< m​onerobull:matrix.org >__ Hello     

> __< plowsof >__ 👋     


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)

**Comment by plowsof** [comment]
Logs 

> __< plowsof >__ Meeting time https://github.com/monero-project/meta/issues/1120     

> __< plowsof >__ greetings all!     

> __< plowsof >__ birthday wishes to RandomX which celebrates its 5th https://www.reddit.com/r/Monero/comments/1h6e4      

> __< l​m:matrix.baermail.fr >__ 404     

> __< s​yntheticbird:monero.social >__ hello, happy birthday randomX     

> __< m​alori:xavi.lu >__ happi birthday  randomX :3     

> __< plowsof >__ sorry, correct link https://www.reddit.com/r/Monero/comments/1h6e4nk/randomx_5_year_anniversary/     

> __< plowsof >__ [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/)     

> __< plowsof >__ any other none haveno-related news since the last meeting? 😅 Revuo looks packed with some recent events      

> __< plowsof >__ rucknium.me/donate is functional again after nioCat reported outdated rates, had to navigate docker with him and then update the backend with latest binaries and such , got there in the end      

> __< plowsof >__ logs of the last MRL meeting @ https://github.com/monero-project/meta/issues/1119     

> __< m​onerobull:matrix.org >__ Hello     

> __< plowsof >__ 👋     

> __< plowsof >__ we can jump into the CCS ideas then if nothing else is to be shared      

> __< plowsof >__ 5. [CCS updates](https://ccs.getmonero.org/)         

> __< plowsof >__   a [v1docq47 - monerotopia 2024 voiceovers and working on xmr.ru](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/517)         

> __< plowsof >__ this seems to have sat long enough now and has obtained the usual positive feedback after being shared around xmr.ru channels      

> __< o​frnxmr:xmr.mx >__ Fcmp dev ccs is complete     

> __< o​frnxmr:monero.social >__ Sorry, slow HS     

> __< s​yntheticbird:monero.social >__ Pretty happy to see the russian community active. too much FOSS places where russians have been expelled     

> __< m​alori:xavi.lu >__ more people knowing and using monero is just a plus     

> __< nioCat >__ s e l s t a     

> __< ofrnxmr >__ atsles is always hete     

> __< ofrnxmr >__ Here*     

> __< ofrnxmr >__ Merge A      

> __< plowsof >__ thanks ofrnxmr, yes, kayabanerves update on FCMP development completion https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/448#note_27450      

> __< plowsof >__ ok so merging v1do moving on      

> __< plowsof >__   b. [selsta part-time monero development (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/522)         

> __< ofrnxmr >__ Merge     

> __< nioCat >__ ++     

> __< a​nchoc:matrix.org >__ +1     

> __< plowsof >__ unsurprisingly support for selsta on the gitlabs / here, merging selsta      

> __< s​yntheticbird:monero.social >__ selsta the GOAT     

> __< s​yntheticbird:monero.social >__ (Greatest Of All Time)     

> __< selsta >__ <3     

> __< s​yntheticbird:monero.social >__ =/= 🐐     

> __< ofrnxmr >__ Selsta can also climb steep cliffs     

> __< plowsof >__   c. Kewbits Haveno GUI [update / partial payout request](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/489#note_27599)     

> __< ofrnxmr >__ skip for now     

> __< ofrnxmr >__ Do the email one first     

> __< plowsof >__ can give kewbit more time to join if he wishes      

> __< ofrnxmr >__ Email one is faster     

> __< plowsof >__   d. [CryptoCheckout Email marketing campaign to onboard new merchants](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/520)     

> __< ofrnxmr >__ Close     

> __< plowsof >__ the project that this wishes to advertise, is going to be closed / not discussed at meetings anymore ( cryptocheckout https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/514 ) so i doubt we want to fund an email campaign for it     

> __< ofrnxmr >__ Were closing the other half of this, i dont see why we'd pay for an email campaign for bad sw     

> __< s​yntheticbird:monero.social >__ I'm not a big fan of CCS endorsing a marketing campaign for a proprietary product     

> __< ofrnxmr >__ +1 plow     

> __< s​yntheticbird:monero.social >__ I vote for closing     

> __< m​onerobull:matrix.org >__ Close     

> __< d​iego:cypherstack.com >__ hi sorry I'm late     

> __< ofrnxmr >__ Better be     

> __< plowsof >__ 👋     

> __< ofrnxmr >__ Ok, we can do kewbit now     

> __< s​yntheticbird:monero.social >__ New reddit post:     

> __< s​yntheticbird:monero.social >__ [Drama] ofrnxmr casually threatening people for not being late.     

> __< plowsof >__ ok closing cyrptocheckout + email campaign     

> __< plowsof >__   c. Kewbits Haveno GUI [update / partial payout request](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/489#note_27599)     

> __< s​yntheticbird:monero.social >__ everyone to their battlestations     

> __< ofrnxmr >__ 1 person at a time 😆     

> __< plowsof >__ so , yes/no for the 50% payout request before source is provided for some miletone(s)     

> __< plowsof >__ can we start there? easier      

> __< d​iego:cypherstack.com >__ obvious no     

> __< ofrnxmr >__ Absolutely not     

> __< a​nchoc:matrix.org >__ no     

> __< s​yntheticbird:monero.social >__ no     

> __< l​m:matrix.baermail.fr >__ no     

> __< m​onerobull:matrix.org >__ No     

> __< plowsof >__ request Denied 🔨 , now, the floor is somewhat open      

> __< s​yntheticbird:monero.social >__ so here's a long ass list in case people forgot:     

> __< s​yntheticbird:monero.social >__ - insult, defamation and behavior during first milestone and beyond     

> __< s​yntheticbird:monero.social >__ - bisq criticism and other clear evidence of the lack of effort put in the CCS.     

> __< s​yntheticbird:monero.social >__ - break of word with Diego     

> __< s​yntheticbird:monero.social >__ - haveno.com domain and website     

> __< s​yntheticbird:monero.social >__ - havenodex on docker hub     

> __< s​yntheticbird:monero.social >__ - haveno crate     

> __< s​yntheticbird:monero.social >__ - the delusion on the exit-scam     

> __< s​yntheticbird:monero.social >__ - lack of documentation on his ai generated shit he delivered in his repo and lied about the deliverable being 'most up to date'.     

> __< s​yntheticbird:monero.social >__ - the Lie regarding `monero-wallet` (he didn't know about it, yet retweeted about it). don't tell he isn't the one that have flagged it.     

> __< s​yntheticbird:monero.social >__ - bitkew grabbing link (doxxing attempt) + monerobull password reset     

> __< d​iego:cypherstack.com >__ Too much drama, immediate closure is necessary. It's taking unnecessary time and space from contributors     

> __< ofrnxmr >__ +1 for closing and ban from ccs and bounty     

> __< s​yntheticbird:monero.social >__ +1 with diego     

> __< l​m:matrix.baermail.fr >__ - not working lib for which he got payed     

> __< d​iego:cypherstack.com >__ No accusations have been addresses on anything from AI to no source with anything of substance. Just whining.     

> __< d​iego:cypherstack.com >__ Life is too short to move forward with this proposal.     

> __< plowsof >__ communities trust has been eroded here, overwhelmingly. there are some positives - the monero-wallet crate name was handed over to boog900 unconditionally after a request...      

> __< ofrnxmr >__ Worse than whining. The issues keep piling up     

> __< m​onerobull:matrix.org >__ At this point I would not be surprised by a backdoor, kewbits code can not be trusted in my opinion     

> __< ofrnxmr >__ Boog voted to close     

> __< plowsof >__ its true that there is not enough bandwidth to discuss only ONE ccs proposal for days at a time in here      

> __< b​oog900:monero.social >__ My crates were then deleted btw     

> __< plowsof >__ what does that mean boog?      

> __< m​onerobull:matrix.org >__ Send the funds to woodser and let him handle it     

> __< plowsof >__ intended? or     

> __< ofrnxmr >__ https://codeberg.org/HavenoDEX/ diego     

> __< b​oog900:monero.social >__ An hour later after the transfer my crates were deleted. I was told someone reported one of my crates. I have since got the cuprate crates back     

> __< m​onerobull:matrix.org >__ Doesn't have to be earmarked for an app, just generally use the money to fund haveno development     

> __< ofrnxmr >__ Mb, thats a new ccs.     

> __< w​oodser:monero.social >__ only thing I'd propose is giving a deadline to release the code, in hopes of getting the code for the greater benefit of the project     

> __< ofrnxmr >__ Woodser https://codeberg.org/HavenoDEX/haveno-app     

> __< ofrnxmr >__ https://codeberg.org/HavenoDEX ***     

> __< l​m:matrix.baermail.fr >__ Did you try the haveno dart package for the interface ?     

> __< d​iego:cypherstack.com >__ The drain on the non-monetary resources of the community far outweighs the benefit of getting some code that will have to be looked over and verified VERY thoroughly.     

> __< ofrnxmr >__ You want to give more time to someone who uses you for their own personal gain?     

> __< s​yntheticbird:monero.social >__ Even if kewbit is supposedly gatekeeping a fully-fledged application, do we want kewbit to be the maintainer?     

> __< plowsof >__ traditionally the CCS does give the proposer ALOT OF LENIENCY in regards to expiry dates (see work in progress ) but this is an exceptional case.     

> __< d​iego:cypherstack.com >__ speaking of, expiry dates are required by the rules, and we should be better about enforcing that     

> __< d​iego:cypherstack.com >__ it would really make this a non-issue     

> __< plowsof >__ i pressume a full audit (cryptographic / security) even would have to be funded due to doubts raising? all of which is not free     

> __< ofrnxmr >__ no. Some ppl have expiry dates that we dont enforce     

> __< ofrnxmr >__ Like soloptxmr     

> __< s​yntheticbird:monero.social >__ I agree. There wasn't any excuses at all, valid or not from kewbit regarding his delays, yet we accepted.     

> __< m​alori:xavi.lu >__ enforcing expiry date is heh, a lot of stuff can happen in one's life that can explain delivering late     

> __< m​alori:xavi.lu >__ as long its delivered in the end     

> __< s​yntheticbird:monero.social >__ Yes that's why I think we should be listening to explanations/ public or private     

> __< ofrnxmr >__ This wouldnt be expired even if there was one.      

> __< plowsof >__ those issues need to be raised / communicated in a timely manner     

> __< d​iego:cypherstack.com >__ We have already taken the high road as a project, this would be taking the road so high it's in the clouds     

> __< plowsof >__ lol     

> __< plowsof >__ angelic music      

> __< a​nchoc:matrix.org >__ the only concern is for other CCS participants. It would be nice for them to have the comfort of knowing an authoritative deadline was presented and every opportunity given. even if discussion is ended immediately and the deadline is short, 48 hours?     

> __< plowsof >__ i see a face in the clouds.. luigi1111 👼     

> __< d​iego:cypherstack.com >__ that said, I'm willing to throw my vote behind woodser     

> __< a​nchoc:matrix.org >__ the angels will thank us     

> __< ofrnxmr >__ This is simply someone who thought they could abuse ccs, made some $, and thinks he can keep pushing      

> __< d​iego:cypherstack.com >__ but I would make the deadline non-generous     

> __< d​iego:cypherstack.com >__ something like a week     

> __< ofrnxmr >__ I'm voting close immediately     

> __< plowsof >__ woodser "only thing I'd propose is giving a deadline to release the code, in hopes of getting the code for the greater benefit of the project"     

> __< s​yntheticbird:monero.social >__ Voting to close immediately too.     

> __< l​m:matrix.baermail.fr >__ Seriously, who went to the work of the first payed milestone and checked if it was even working ?     

> __< s​yntheticbird:monero.social >__ I think aside the enforcement of delays, the verification  of deliverables should also be enforced, at least if there is amendment of rule 4 on update on a regular basis     

> __< d​iego:cypherstack.com >__ Not I. I was assured it met the requirements by several parties, and operated on this assumption. A mistake, clearly.     

> __< ofrnxmr >__ Woodser vouched then too     

> __< plowsof >__ i noted that 76~ payouts have occured since july2nd, with no such squabbles or issues - trust and taking peoples word for things plays a role     

> __< m​onerobull:monero.social >__ Never ever allow compromises on the development must be Foss rule     

> __< m​onerobull:monero.social >__ Never again     

> __< m​alori:xavi.lu >__ +1     

> __< s​yntheticbird:monero.social >__ I agree with monerobull. Stealing of code in early development process has never been an issue and have always be met with anger and bad outcomes for the stealers     

> __< d​iego:cypherstack.com >__ rule 4 should be amended to read that WIP must be made available upon request from Core regardless of state?     

> __< ofrnxmr >__ https://codeberg.org/HavenoDEX/     

> __< p​reland:monero.social >__ There should also be something along the lines of “I understand what this CCS entails in it’s entirety” in case there’s ever an issue where someone just didn’t read things well enough     

> __< d​iego:cypherstack.com >__ even if it's just to a few people that core can assign to take a look at it and vouch that it exists     

> __< a​nchoc:matrix.org >__ +1     

> __< d​iego:cypherstack.com >__ I know several, including ofrn, have been against all WIP code being available all the time as draconian     

> __< d​iego:cypherstack.com >__ and I can see this viewpoint     

> __< p​reland:monero.social >__ There should also be some form of a continuation plan in case someone has to stop their CCS before it is completed (ie this situation, or someone just keels over and dies in the middle of theirs)     

> __< s​yntheticbird:monero.social >__ diego if core is ready to endorse the verification and engage its credibility then yes it is reasonable. I still think FOSS at all time would be a simpler and more effective solution     

> __< ofrnxmr >__ no, i think rule 4 should only allow requests for source when the proposer is withholding after periods of timr      

> __< d​iego:cypherstack.com >__ but core should reserve the right to be able to see hidden code for the sake of verification     

> __< d​iego:cypherstack.com >__ at the least because if you know they're not really working on it, that we can do something else with our time and money     

> __< plowsof >__ The CCS (Donors/Community) are the A side in this transaction - there is no "leverage" to be gained from threatening to not release sources - the only damage done is to the proposer himself.      

> __< ofrnxmr >__ Like soloptxmr or e n d o g e n i c     

> __< d​iego:cypherstack.com >__ that said, what rule 4 should be amended to is a bit off-topic on this point of discussion.     

> __< ofrnxmr >__ If you want to pretend like there is a WIP and continuously stall, then you should have tk make that WIP public     

> __< d​iego:cypherstack.com >__ It seems the current topic has a loose consensus of closing the proposal     

> __< plowsof >__ a flat 6 month expiry merge request can be made to soloptxmr      

> __< s​yntheticbird:monero.social >__ its not even loose consensus at this point we're a hive mind     

> __< l​m:matrix.baermail.fr >__ It doesn't seem to work.      

> __< l​m:matrix.baermail.fr >__ https://github.com/KewbitXMR/flutter_haveno/issues     

> __< plowsof >__ there is not much demand to get that finished, currently, only the thorn in my side      

> __< l​m:matrix.baermail.fr >__ the real question from my point of view is not if kewbit is a scammer. He is since he requested funds for unfinished work.     

> __< l​m:matrix.baermail.fr >__ The real question is how does the CCS took the decision to pay the 75xmr for getting a non working library and what to fix to prevent this to happen again ?     

> __< ofrnxmr >__ Soloptxmr had an expiry of oct 2022     

> __< l​m:matrix.baermail.fr >__ Maybe CCS proposals should include fund for testers ?     

> __< l​m:matrix.baermail.fr >__ It doesn't seem to work.      

> __< l​m:matrix.baermail.fr >__ https://github.com/KewbitXMR/flutter\_haveno/issues     

> __< ofrnxmr >__ LM, we took woodsers word at face value     

> __< l​m:matrix.baermail.fr >__ the real question from my point of view is not if kewbit is a scammer. He is since he requested funds for unfinished work.     

> __< l​m:matrix.baermail.fr >__ The real question is how does the CCS took the decision to pay the 75xmr for getting a non working library and what to fix to prevent this to happen again ?     

> __< l​m:matrix.baermail.fr >__ Maybe CCS proposals should include a fund for testers ?     

> __< s​yntheticbird:monero.social >__ Louis-Marie Diego explained he received confirmation by several third parties that the code was ok     

> __< p​reland:monero.social >__ Sad day for woodser     

> __< plowsof >__ we are lucky that the monero-core dev proposals require pull requests to be merged into monero-core repositories .. and their peers are trusted persons who they work around on a day-to-day basis       

> __< ofrnxmr >__ plowsof, another example is jbermans fcmp work     

> __< l​m:matrix.baermail.fr >__ Sorry woodser, did you tried the dart package or just looked at the code ?     

> __< d​iego:cypherstack.com >__ in the early days of the drama, the investigation that was assigned to me wasn't that the code worked, that seemed to be a given, but all that ancillary stuff.     

> __< s​yntheticbird:monero.social >__ Ah alright I understand     

> __< plowsof >__ i looks like a duck - quaks like a duck - but ideally needs an app created that uses it to prove its real  - which is another milestone : this also played a role in decisions      

> __< ofrnxmr >__ the 75xmr was requested to be paid out by woodser, and without any objections from community, it seemed everything was in order for that specific milestone     

> __< plowsof >__ he had more to lose than to gain (still does)     

> __< plowsof >__ (thanks for the vote rbrunner7)     

> __< w​oodser:monero.social >__ I only reviewed the dart package from a high level, and tested the app as an executable to verify the overall thing. I was upfront about this in my feedback for his payout     

> __< s​yntheticbird:monero.social >__ No need for witch hunting, the responsibility is on no one, we all missed diligence regarding kewbit despite his behavior. Dart isn't exactly a mainstream language and we all assume we have better things to do than auditing an in progress code.     

> __< ofrnxmr >__ Kewbit took this as "nobody is looking, i can get away with anything. Even woodser has my back" and proceeded to take advantage of woodsers kindness with stuff like https://codeberg.org/HavenoDEX/     

> __< p​reland:monero.social >__ Since it is looking like the source code won’t be released, I’ll post the screenshots of my conversation with who I now believe was indeed kewbit if wanted     

> __< s​yntheticbird:monero.social >__ ofrnxmr fwiw we can contact codeberg and get the Account back     

> __< ofrnxmr >__ Kewbit also claimed (a few hours ago) to own the rights to haveno name     

> __< s​yntheticbird:monero.social >__ link please     

> __< p​reland:monero.social >__ That reminds me, kewbit was trying to invite me to some Haveno corporate org on GitLab     

> __< l​m:matrix.baermail.fr >__ What I want to point out is not that someone overlooked the code, but that maybe there should be a more written process to verifiy work of milestone.     

> __< s​yntheticbird:monero.social >__ agree, will be something to discuss another time tho     

> __< ofrnxmr >__ https://matrix.to/#/!UuoWQWMGvPBHkIgWGJ:monero.social/$AewU-mVyguXn5Zsqg_VEMnHfnpzQcWT_qi5irVNiU9s?via=xmr.mx&via=monero.social&via=matrix.org     

> __< ofrnxmr >__ "I also own Haveno.Exchange trademark "     

> __< d​iego:cypherstack.com >__ alright. Consensus achieved here.     

> __< plowsof >__ touch crowd      

> __< plowsof >__ tough*     

> __< d​iego:cypherstack.com >__ Next point plowsof     

> __< plowsof >__ not much else .. basically:     

> __< plowsof >__ not discussing / voted to close:     

> __< plowsof >__ e. [Carrot animated video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/506)         

> __< plowsof >__ f. [CryptoCheckout WordPress plugin (for WooCommerce) & Shopify app](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/514)     

> __< plowsof >__ and Any other business really      

> __< ofrnxmr >__ outside of the ccs, its important to note that kewbit is doing all sorts of unethical business against monero contributors     

> __< plowsof >__ if only we knew someone with dart/flutter UI experience in the crypto sphere      

> __< ofrnxmr >__ for an external entity to do these things, we'd assume a bad actor is attacking us. But this is from a ccs proposer who is supposed to be building software for haveno     

> __< ofrnxmr >__ Cake and stack have dart/flutter devs     

> __< s​yntheticbird:monero.social >__ \* Insert gigantic wink wink to CypherStack \*     

> __< ofrnxmr >__ I assume mrcyjanek as well     

> __< d​iego:cypherstack.com >__ ok I admit. kewbit is my alt. I manufactured this whole thing to get Cypher Stack on a CCS retainer to confirm CCSs     

> __< p​reland:monero.social >__ I’m willing to learn stuff (but not until I’m done with my pre-existing work, I can feel the weight of my procra-sins-ation)     

> __< s​yntheticbird:monero.social >__ 🤯.     

> __< s​yntheticbird:monero.social >__ 5D chess     

> __< ofrnxmr >__ cypherstack will build the new app /s     

> __< ofrnxmr >__ And will audit it     

> __< ofrnxmr >__ Also, from my conversation with "markus", i'm pretty sure kewbit is markus is aloha     

> __< m​onerobull:monero.social >__ I gotta go, good meeting thx plwsf     

> __< s​yntheticbird:monero.social >__ thanks mbll     

> __< plowsof >__ thanks all for attending 👍 thanks mbll     

> __< ofrnxmr >__ thanks mbll     

> __< d​iego:cypherstack.com >__ thanks mbll     

> __< p​reland:monero.social >__ “It is with great reluctance that I have agreed to this calling”     

> __< d​iego:cypherstack.com >__ ok well if that's most of the meeting     

> __< d​iego:cypherstack.com >__ then I'll also hop off     

> __< d​iego:cypherstack.com >__ thanks  guys     

> __< p​lowsof:matrix.org >__ 👋     

> __< ofrnxmr >__ For the future, i think anybody on ccs must not take advantage of the community      

> __< s​yntheticbird:monero.social >__ ofrnxmr dropping hot take     

> __< plowsof >__ all future discussion of haveno  gui - core - official - dex will be moved to haveno related channels, no ones interested      

> __< plowsof >__ in seeing the same things commented or "Get fucked"      

> __< r​eal_glitch:matrix.org >__ i dont know whats going on but as the most senior member of this community     

> __< r​eal_glitch:matrix.org >__ i approve the result of this meeting!     

> __< o​frnxmr:xmr.mx >__ https://matrix.to/#/#haveno.exchange:monero.social     

> __< nioCat >__ I was not here for the Qbit discussion         

> __< nioCat >__ a comment to those that are relatively new          

> __< nioCat >__ the CCS usually has a requirement that the person has previously contributed to monero in order to "prove" themselves          

> __< nioCat >__ in the form of work done voluntarily      

> __< o​frnxmr:xmr.mx >__ i think that's a fair requirement. Stops random outsiders from opening "checkout/email campaign" ccs     

> __< a​nchoc:matrix.org >__ i like that, is it an informal requirement? i don't see it ccs.getmonero.org/what-is-ccs/     

> __< o​frnxmr:xmr.mx >__ Yes     

[ ... ]

> __< o​frnxmr:xmr.mx >__ niocat plowsof << should we make it a formal requirement?     

Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)
